lib/shallow_attributes/class_methods.rb

```diff
@@ -6,6 +6,19 @@ module ShallowAttributes
   #
   # @since 0.1.0
   module ClassMethods
+    # Inject our default values into subclasses.
+    #
+    # @private
+    #
+    # @param [Object] subclass
+    #
+    def inherited(subclass)
+      super
+      if respond_to?(:default_values)
```

Comment:
Given `ShallowAttributes` is intended to be a mixin we can't be certain that the base class will implement `default_values`. That's why it's necessary for us to check first.